src/components/elements/accordion/accordion.tsx

```diff
@@ -15,14 +15,14 @@ export const Accordion = (props: IGenericElementProps<IAccordionSchema>) => {
 	// CONST, STATE, REF
 	// =============================================================================
 	const { schema, id } = props;
-	const { children, button, title, disableContentInset, ...otherSchema } = schema;
+	const { children, button, title, titleSchema, disableContentInset, ...otherSchema } = schema;
 
 	const { dispatchFieldEvent } = useFieldEvent();
 
 	return (
 		<BoxContainer
 			id={id}
-			title={title}
+			title={titleSchema ? <Wrapper id={id}>{titleSchema}</Wrapper> : title}
```

Comment:
just for correctness, avoid passing in the same id on the child as that might result in a duplicate id on the title (for DS components that consume the id prop)

```suggestion
			title={titleSchema ? <Wrapper>{titleSchema}</Wrapper> : title}
```

Reply:
udpated